registration/templates/registration/admin_approve_complete_email.txt

```diff
@@ -0,0 +1,13 @@
+{% load i18n %}
+{% blocktrans %}
+Your account is now approved. You can log in using the following link
+{% endblocktrans %}
+http://{{site.domain}}{% url 'django.contrib.auth.views.login' %}
```

Comment:
The test failures on 1.10 are because reversing by dotted path is removed. These templates should reverse by URL name (e.g. 'login') instead of by dotted path.